llpc/context/llpcCompiler.cpp

```diff
@@ -2446,7 +2452,7 @@ Result Compiler::BuildRayTracingPipeline(const RayTracingPipelineBuildInfo *pipe
     }
 
     pipelineOut->pipelineBinCount = elfBinarys.size();
-    BinaryData *pipelineBins = reinterpret_cast<BinaryData *>(voidPtrInc(allocBuf, elfSize));
+    BinaryData *pipelineBins = voidPtrIncAligned<BinaryData>(allocBuf, elfSize);
```

Comment:
You can't just do this without also adjusting the allocation size that's computed above.

Reply:
Can you please add a comment as well to describe the possibility of UB without your change?